pkgs/development/python-modules/fastdtw/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "fastdtw";
+  version = "0.3.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2350fa6ec36bcad186eaf81f46eff35181baf04e324f522de8aeb43d0243f64f";
+  };
+
+  nativeBuildInputs = [
+    cython
+    numpy
+  ];
+
+  # Pypi's tarball doesn't contain tests
+  doCheck = false;
+  pythonImportsCheck = [ "fastdtw.fastdtw" ];
+
+  meta = {
+    description = ''
+      Python implementation of FastDTW.
+      FastDTW is an approximate Dynamic Time Warping (DTW) algorithm that provides
+      optimal or near-optimal alignments with an O(N) time and memory complexity.
+    '';
```

Comment:
description should be a one line summary. For longer descriptions use `longDescription`